packages/pie/src/PieLegends.tsx

```diff
@@ -23,7 +23,7 @@ const PieLegends = <RawDatum,>({
                     {...legend}
                     containerWidth={width}
                     containerHeight={height}
-                    data={dataWithArc}
+                    data={legend.data ?? dataWithArc}
```

Comment:
As [we already have some unit tests for the legends](https://github.com/plouc/nivo/blob/master/packages/pie/tests/Pie.test.tsx#L518), I think we could add a test case for this, what do you think?

Reply:
Oh, yes definitely will add a test. Good catch.

Reply:
Thanks!